ci/citest/rpmbuild/el7.Dockerfile

```diff
@@ -1,9 +1,11 @@
 FROM centos:7
 WORKDIR /var/tmp
 ENTRYPOINT ["/sbin/init"]
+RUN yum install -y yum-utils
```

Comment:
I don't really see why you'd make an extra layer of docker images by installing this here but I'm not gonna hold back the merge for that.